Packs/ThreatZone/Integrations/ThreatZone/ThreatZone.py

```diff
@@ -0,0 +1,544 @@
+            # data = client.threatzone_get_sanitized(submission_uuid)
+            # f_res = fileResult(f"sanitized-{submission_uuid}.zip", data)
+            res = create_res(readable_dict, output)
+            # res.append(f_res)
+
+    except Exception as e:
+        output = {'ThreatZone.Result(val.Job_ID == obj.Job_ID)': {'REPORT': result}}
+        res = create_res(result, output, exception=e)
+    return res
+
+
+def threatzone_check_limits(client: Client) -> CommandResults:
+    availability = client.threatzone_check_limits(None)
+    readable_output = tableToMarkdown('LIMITS', availability["Limits"])
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_key_field='results',
+        outputs={'ThreatZone.Limits(val.Job_ID == obj.Job_ID)': availability["Limits"]}
+    )
+
+
+def threatzone_sandbox_upload_sample(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Uploads the sample to the ThreatZone sandbox to analyse with required or optional selections."""
+    availability = client.threatzone_check_limits("sandbox")
+    if not availability["avaliable"]:
+        raise DemistoException(
+            f"Reason: {availability['Reason']}\nSuggestion: {availability['Suggestion']}\nLimits: {availability['Limits']}")
+
+    ispublic = args.get('isPublic')
+    environment = args.get('environment')
+    work_path = args.get('work_path')
+    timeout = args.get('timeout')
+    mouse_simulation = args.get('mouse_simulation')
+    https_inspection = args.get('https_inspection')
+    internet_connection = args.get('internet_connection')
+    raw_logs = args.get('raw_logs')
+    snapshot = args.get('snapshot')
+    file_id = args.get('entry_id')
+    file_obj = demisto.getFilePath(file_id)
+    file_name = encode_file_name(file_obj['name'])
+    file_path = file_obj['path']
+
+    files = [
+        ('file', (file_name, open(file_path, 'rb'), 'application/octet-stream'))
+    ]
+
+    param = {
+        'scan_type': "sandbox",
+        'environment': environment,
+        'private': ispublic,
+        'timeout': timeout,
+        'work_path': work_path,
+        'mouse_simulation': mouse_simulation,
+        'https_inspection': https_inspection,
+        'internet_connection': internet_connection,
+        'raw_logs': raw_logs,
+        'file_path': file_path,
+        'snapshot': snapshot,
+        'files': files
+    }
+
+    result = client.threatzone_add(param=param)
+    readable_output = tableToMarkdown('SAMPLE UPLOADED', result)
+    uuid = result['uuid']
+    url = f"https://app.threat.zone/submission/{uuid}"
+    availability = client.threatzone_check_limits("sandbox")
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_key_field='results',
+        outputs={'ThreatZone.Limits(val.Job_ID == obj.Job_ID)': availability["Limits"],
+                 'ThreatZone.Analysis(val.Job_ID == obj.Job_ID)': {'UUID': uuid, 'URL': url}}
+    )
```

Comment:
as mention above - please refactor the usage of CommandResults, and if you need to return two different keys to the context (like `ThreatZone.Limit` and `ThreatZone.Analysis`) you can return list of two CommandResults one for each of them